src/Plugin_Command.php

```diff
@@ -748,6 +758,57 @@ protected function get_item_list() {
 		return $items;
 	}
 
+	/**
+	 * Get the wordpress.org status of a plugin.
+	 *
+	 * @param string $plugin_name The plugin slug.
+	 *
+	 * @return string The status of the plugin, includes the last update date.
+	 */
+	protected function get_wporg_data( $plugin_name ) {
+		$data = [
+			'status'       => 'no_wp_org',
+			'last_updated' => '-',
```

Comment:
```suggestion
			'last_updated' => '',
```

We can leave this empty when it's not present.